tests/test_cases/fstatfs.c

```diff
@@ -0,0 +1,37 @@
+#undef _GNU_SOURCE
+#define _GNU_SOURCE
+
+#include <errno.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/statfs.h>
+
+int main(int argc, char **argv) {
+  // int fd = open("/dev/urandom", O_RDONLY, S_IRWXU|S_IRWXG|S_IRWXO);
```

Comment:
Can remove this comment if not needed.

Reply:
I set this line to test whether fstatfs works for system file

Reply:
yes please remove this comment